functions.php

```diff
@@ -56,4 +58,25 @@ function displayPokemon(array $pokemons): string {
         }
     }
     return $result;
-}
\ No newline at end of file
+}
+
+/**
+ * Adds data to the database
+ *
+ * @param none
+ *
+ * @return array, when returned the data will be sent to the database
+ */
+function addPokemonToDatabase()
+{
+    $dataBase = getDatabaseObject();
+    $pokedexNumber = $_POST['addPokedexNo'];
```

Comment:
what happens if there is no data in the post? or no data in the `addPokedexNo` field?

Reply:
there can't be as they are required fields and will not allow you to submit without them

Reply:
it breaks just tested it i thought it still had required tag

Reply:
you cannot rely on the HTML required tag, it is VERY easy to get around (just remove it in the dev tools), you ALWAYS have to validate on the backend. dont trust the users!

Reply:
updated!